waterbox/bsnescore/bsnes/target-bsnescore/bsnescore.cpp

```diff
@@ -122,32 +122,41 @@ EXPORT void snes_set_callbacks(SnesCallbacks* callbacks)
     snesCallbacks = SnesCallbacks(*callbacks);
 }
 
-EXPORT void snes_init(int entropy, uint left_port, uint right_port, uint16_t merged_bools)// bool hotfixes, bool fast_ppu)
+struct SnesInitData {
+    int entropy;
+    uint left_port;
+    uint right_port;
+    int hotfixes;
+    int fast_ppu;
```

Comment:
Any reason for making these `int` instead of `bool`?

Reply:
/shrug

Reply:
Well apparently it breaks as bool, fun